docs/concepts/controller-manager.md

```diff
@@ -41,6 +41,12 @@ Consequently, to ensure that `CloudProfile`s in-use are always present in the sy
 This controller consists out of three reconciliation loops:
 The main loop is reconciling `Project` resources while the second loop is controlling the necessary actions for stale projects.
 
+### `Quota` Controller
```

Comment:
This section is not correctly placed: the next subsections (`####`) belong to the `Project` controller.
Please move this section down.